docs/lib/auth/fragments/flutter/signin_web_ui/20_platform_specific_setup.md

```diff
@@ -0,0 +1,45 @@
+## How It Works
+Signin with web UI requires the Amplify plugin to display the signin UI inside a webview. This is managed internally by amplify-android and amplify-ios. After the signin process is complete they will redirect back to your app.
```

Comment:
Not sure if we should add the following

>This is managed internally by amplify-android and amplify-ios. After the signin process is complete they will redirect back to your app

Reply:
![image](https://user-images.githubusercontent.com/10621276/105774471-246f6280-5f1a-11eb-9881-6a796cb764df.png)

Reply:
I edited as we discussed.